#! /bin/sh /usr/share/dpatch/dpatch-run
## no-xgpsspeed-manpage.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: we don;t build xgpsspeed, so don't mention it in the manapge

@DPATCH@
diff -urNad gpsd~/gps.xml gpsd/gps.xml
--- gpsd~/gps.xml	2006-12-15 00:43:39.000000000 +0100
+++ gpsd/gps.xml	2007-10-14 14:53:17.000000000 +0200
@@ -11,7 +11,6 @@
 <refnamediv id='name'>
 <refname>gps</refname>
 <refname>xgps</refname>
-<refname>xgpsspeed</refname>
 <refname>cgps</refname>
 <refname>cgpxlogger</refname>
 <refpurpose>test clients for gpsd</refpurpose>
@@ -37,22 +36,6 @@
       </group>
 </cmdsynopsis>
 <cmdsynopsis>
-  <command>xgpsspeed</command> <arg choice='opt'>-rv</arg> 
-      <arg choice='opt'><replaceable>X-options</replaceable></arg>
-      <arg choice='opt'>-h </arg> 
-      <arg choice='opt'>-V </arg> <arg
-      choice='opt'>-nc <replaceable>X-color</replaceable></arg> <arg
-      choice='opt'>-speedunits <group
-      choice='req'><arg>mph</arg><arg>kph</arg><arg>knots</arg></group></arg>
-      <group>
-	<replaceable>server</replaceable>
-	  <group> 
-	    <replaceable>:port</replaceable> 
-	      <group><replaceable>:device</replaceable></group>
-	  </group>
-      </group>
-</cmdsynopsis>
-<cmdsynopsis>
   <command>cgps</command>  
       <arg choice='opt'>-h </arg>
       <arg choice='opt'>-j </arg>
@@ -130,21 +113,6 @@
 12 satellites.</para>
 
 </refsect2>
-<refsect2><title>xgpsspeed</title>
-
-<para><application>xgpsspeed</application> is a speedometer that uses
-position information from the GPS. It accepts an -h option and
-optional argument as for <application>gps</application>, or a -V
-option to dump the package version and exit.  Additionally, it accepts
--rv (reverse video) and -nc (needle color) options.</para>
-
-<para>The -speedunits option can be used to set the speed units for
-display; follow the keyword with knots for nautical miles per hour,
-kph for kilometres per hour, or mph for miles per hour.  The default
-is miles per hour.   This option can also be set as the X resource 
-'speedunits'.</para>
-
-</refsect2>
 <refsect2><title>cgps</title>
 
 <para><application>cgps</application> is a client resembling 
